src/System.Text.Json/tests/JsonElementWriteTests.cs

```diff
@@ -1192,13 +1158,6 @@ private static void WriteSimpleValue(bool indented, string jsonIn, string jsonOu
                 writer.WriteEndObject();
                 writer.Flush();
 
-                if (indented && s_replaceNewlines)
-                {
-                    AssertContents(
```

Comment:
Are we losing any coverage when removing these?

Reply:
no, this is already checked - in some cases we have previously double checked the same assert